generator/integration_tests/golden/tests/golden_kitchen_sink_stub_test.cc

```diff
@@ -21,8 +21,7 @@
 
 using ::google::cloud::testing_util::IsOk;
 using ::google::cloud::testing_util::StatusIs;
-using ::testing::_;
-using ::testing::ByMove;
+using ::testing::_;  // NOLINT(bugprone-reserved-identifier)
```

Comment:
I wonder why we don't need the NOLINT on other such incantations.

Reply:
This one is in the global namespace, most others are in `google::cloud::*::` and then some anonymous namespace.

Reply:
Consider moving all these using declarations line 32 or so.